core/src/Streamly/Internal/Data/Unboxed.hs

```diff
@@ -14,18 +14,21 @@ module Streamly.Internal.Data.Unboxed
     , getMutableByteArray#
     , pin
     , unpin
-    , newUnpinnedArrayContents
+    , newUnpinnedBytes
+    , newAlignedPinnedBytes
+    , nil
```

Comment:
this module is looking more like MutableByteArray module.

Reply:
I will rename it in another PR.

Reply:
See https://github.com/composewell/streamly/issues/1910